BUILDING.md

```diff
@@ -163,7 +163,8 @@ $ [sudo] make install
 
 Prerequisites:
 
-* [Python 2.6 or 2.7](https://www.python.org/downloads/)
+* [Python 2.6 or 2.7](https://www.python.org/downloads/)  
+  *Note*: It is recommanded to include the "Add python.exe to path" option
```

Comment:
> add a note about the alternative ("add it to %Path% manually")

🤔 I don't like "spoon feeding" devs, so I need to think about this.
P.S. The installer __will__ register the installation so #13882 should cover that. (To avoid that I have a special tool that extracts `msi` files so I can xcopy the extracted files to where I like, and then I do what's necessary for my system to work.)